backend/attendance-service/Attendance-Service-1.0.0.yaml

```diff
@@ -484,10 +493,6 @@ components:
         userId:
           type: string
           description: Identifier of the user of the system
-        permissionLevels:
```

Comment:
Don't remove the permission level just yet. Let me discuss this with Subhashini and finalize.